prawcore/sessions.py

```diff
@@ -33,6 +35,7 @@ class Session(object):
                          codes['gateway_timeout']: ServerError,
                          codes['internal_server_error']: ServerError,
                          codes['not_found']: NotFound,
+                         codes['read_timeout']: ReadTimeout,
```

Comment:
This `ReadTimeout` refers to `requests.exceptions.ReadTimeout`. How things are imported will need to be changed so that this maps to `prawcore.exceptions.ReadTimeout`.

Another option is to use a more general exception name in prawcore like `NetworkException` instead of the more specific `ReadTimeout`, then there won't be a naming conflict as well.

Reply:
Oh right forgot about that import. Good idea with the general network exception in any case. Just changed things to reflect that.

Thanks for the patience and the guidance, as you may have guessed, this is my first PR to an open source project. :-)